junk-drawer/tweens.janet

```diff
@@ -176,7 +176,7 @@ examples/07-tweens.janet for more!
              :table (merge (current key) val))))
 
     # Tick the tweens elapsed time, delete it if we've reached its duration
-    (if-let [new-elapsed (+ elapsed 1)
-             complete? (> new-elapsed duration)]
-      (remove-entity wld tween-ent)
-      (put tween-data :elapsed-time new-elapsed))))
+    (let [new-elapsed (+ elapsed 1)]
+      (if (>  elapsed  duration)
```

Comment:
theres a test that's failing I _think_ because the move from `if-let` has introduced a off by one difference. Change this line to fixes fixes the test. 

```suggestion
      (if (>=  elapsed  duration)
```

though I'm still thinking through why that change would be needed :thinking: